### PR TITLE
Fix crash in generate constructor when type declaration is improperly…

### PR DESCRIPTION
**Pull request by jmarolf**
… formed

Fixes https://github.com/dotnet/roslyn/issues/6918

### CODE REVIEW

**Comment by jmarolf** [comment]
@dotnet/roslyn-ide

File: src/Features/VisualBasic/Portable/GenerateMember/GenerateConstructor/VisualBasicGenerateConstructorService.vb [hunk 1]
```diff
@@ -163,8 +163,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.GenerateMember.GenerateConstructor
             Dim classBlock = DirectCast(classDeclaration, ClassBlockSyntax)
             Dim classSymbol = semanticModel.GetDeclaredSymbol(classBlock.BlockStatement, cancellationToken)
 
-            Dim baseType = classSymbol.BaseType
-            Dim constructor = baseType.Constructors.FirstOrDefault(Function(c) IsSymbolAccessible(c, document))
+            Dim baseType = classSymbol?.BaseType
```
Comment (brettfo) [inline src/Features/VisualBasic/Portable/GenerateMember/GenerateConstructor/VisualBasicGenerateConstructorService.vb]: Inline `baseType`?

**Comment by brettfo** [comment]
:shipit:

**Comment by Pilchie** [comment]
:+1:

**Comment by dpoeschl** [comment]
:+1:

**Pull request merged**